plugins/awards-backend/README.md

```diff
@@ -58,6 +58,23 @@ function makeCreateEnv(config: Config) {
 }
 ```
 
+## Configuration
+
+### Slack notifications
+
+To enable Slack notifications, add the following to your `app-config.yaml` file:
+
+```yaml
+awards:
+  notifications:
+    slack:
+      webhook:
+        # https://api.slack.com/messaging/webhooks
+        url: <my_slack_webhook_url>
```

Comment:
Doesn't the webhook url contain a secret? i think in the documentation here we should advise that it be included as an env var and referenced as such (from what I understand about dynamic includes in backstage)

Reply:
done here [8d2d2b2](https://github.com/seatgeek/backstage-plugins/pull/31/commits/8d2d2b2e7201ce706d3b7b91ce1486e943d6ff12)